test/unit/testEligibilityValidationStep.js

```diff
@@ -28,6 +28,28 @@ describe('EligibilityValidationStep', () => {
             res = {};
         });
 
+        it('should set feature toggles in the ctx correctly', (done) => {
```

Comment:
This test is duplicating `a GET request should not set ctx.answerValue when an answer value is not given and call eligibilityCookie.getAnswer()` and the testing for the toggles would probably be better in that test.

Reply:
Done